service/deploy.py

```diff
@@ -76,6 +76,43 @@ def ready_to_deploy(instance_ip, username, instance_id):
         limit_playbooks=['check_networking.yml'])
 
 
+def deploy_mount_volume(instance_ip, username, instance_id,
+        device, mount_location=None, device_type='ext4'):
+    """
+    Use service.ansible to mount volume to an instance.
+    """
+    extra_vars = {
+        "VOLUME_DEVICE": device,
+        "VOLUME_MOUNT_LOCATION": mount_location,
+        "VOLUME_DEVICE_TYPE": device_type,
+    }
+    playbooks_dir = settings.ANSIBLE_PLAYBOOKS_DIR
+    playbooks_dir = os.path.join(playbooks_dir, 'utils')
+    limit_playbooks = ['mount_volume.yml']
+    return ansible_deployment(
+        instance_ip, username, instance_id, playbooks_dir,
+        limit_playbooks=limit_playbooks,
+        extra_vars=extra_vars)
+
+
+def deploy_check_volume(instance_ip, username, instance_id,
+        device, device_type='ext4'):
+    """
+    Use service.ansible to deploy to an instance.
```

Comment:
I'm not sure what you mean by "deploy to an instance".. this function appears to just run check_volume.yml against the instance?